cmd/antrea-controller/controller.go

```diff
@@ -261,6 +263,11 @@ func run(o *Options) error {
 		traceflowController = traceflow.NewTraceflowController(crdClient, podInformer, tfInformer)
 	}
 
+	var packetSamplingController *packetsampling.Controller
+	if features.DefaultFeatureGate.Enabled(features.PacketSampling) {
+		packetSamplingController = packetsampling.NewPacketSamplingController(crdClient, podInformer, psInformer)
+	}
```

Comment:
I thought we agree that PacketSampling doesn't need a global unique tag, right? Why we still need antrea-controller to allocate tag and maintain the tag value in status? Feel free to reach me directly to discuss if it's hard to explain via comments.

Reply:
We can use tag to differentiate sessions. My point is we don't need a **globally unique** tag. We just need to allocate one tag locally when handling a packetsampling object in agent, right? Exposing such implementation detail to user-facing API and introducing a controller in antrea-controller unnecessarily is what I want to avoid. IMO it's just like the NetworkPolicy openflow rule ID, or the OVS port ID, which is essentially internal stuff and means nothing to users.

Reply:
Ok. Will refactor this part to remove the controller.